README.md

```diff
@@ -72,7 +72,6 @@ TENANT_TOKEN="<INSERT-TOKEN-FROM Hosted Mender>"
             --raw-disk-image $RAW_DISK_IMAGE                \
             --mender-disk-image $MENDER_DISK_IMAGE          \
             --device-type $DEVICE_TYPE                      \
-            --mender-client /mender                         \
```

Comment:
What about adding brackets to emphasize it is optional argument?

Reply:
This part in the README is meant for users to copy/paste it. I don't think it is a good idea with brackets here.